docs/UserGuide.md

```diff
@@ -108,74 +108,56 @@ The list of companies should be listed in the following format below:
 ```
 ### Deleting a company : `delete` ###
 
-Deletes a company from the address book.
-
 **Format:`delete INDEX`**
 
-* Deletes the contact at the specific INDEX
-* The index refers to the index number shown in the displayed contact list
+You add an entry by mistake or you no longer want to track a particular company, 
+you can delete it using the delete command. Which deletes the company at the specified INDEX.
```

Comment:
Phrasing is slightly weird with minor grammatical errors. 

One suggestion I could think of:
Whether you accidentally added a company or no longer want to track it, don't worry – our system's got you covered! Just use the command `delete` to delete the company from the list. 

Since we're including the Format and Examples already, I don't think we have to describe the index in the description. 

Also, have the description before Format